tests/integration/docker/swarm.bats

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Swarm testing
+
+SRC="${BATS_TEST_DIRNAME}/../../integration/lib/"
+number_of_replicas=12
+
+setup() {
+	source $SRC/common.bash
+	cleanDockerPs
+	runtimeDocker
+	$DOCKER_EXE swarm init
+}
+
+@test "check that the replicas' names are different" {
+	$DOCKER_EXE service create --name testswarm --replicas $number_of_replicas --publish 8080:80 mcastelino/nginx /bin/bash -c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\"" 
```

Comment:
@jodh-intel eventually we should switch to upstream ngnix or I can create an auto built image on docker hub... I think @GabyCT ran into issues with the upstream image. I will look into the root cause for the upstream failure.

Alternately we can just generate the image on the fly. But it will not scale when we run this test on a true cluster. I will work on a local swarm test (which is all we are doing today anyway).